qmcpy/discrete_distribution/halton.py

```diff
@@ -4,6 +4,7 @@
 from numpy import *
 from .c_lib import c_lib
 import ctypes
+from math import *
```

Comment:
what is this used for?

Reply:
It is used for the ceiling function

Reply:
Sounds good. `numpy.ceil` works if you need it vectorized, but either way is fine.